great-idea-website/css/index.css

```diff
@@ -63,4 +63,92 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+body{
+	padding:200px;
+}
+
+
+a {
+display:inline-block;
+color:gray;
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+text-decoration:none;
+margin:30px 35px;
+}
+
+img {
+	display:inline;
+}
+
+div{
+	width:49%;
+	display:inline-block;
+	text-align:center;
+	font-size:100px;
+	
+}
+
+button{
+	border:1px solid black;
+	padding:10px 50px;
+	margin-bottom: 40%;
```

Comment:
I'll advise that you add bottom margin to the container wrapping the button and text. Easier to style